src/resources/elements/tokenDistribution/tokenDistribution.ts

```diff
@@ -0,0 +1,25 @@
+import { ITokenDetails } from "../../../newLaunch/launchConfig";
+import { bindable } from "aurelia-framework";
+import "./tokenDistribution.scss";
+import * as moment from "moment";
```

Comment:
Can our `DateService` service the purposes here, or can we benefit by adding to it any new and useful functionality found here?